src/commoncode/fetch.py

```diff
@@ -39,6 +39,11 @@
 # logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
 # logger.setLevel(logging.DEBUG)
 
+def test_download(url):
```

Comment:
Would you mind call this check_download? Also what's the value to download and discard?